content/en/getting_started/containers/datadog_operator.md

```diff
@@ -62,6 +62,39 @@ The [Datadog Operator][1] is an open source [Kubernetes Operator][2] that enable
   kubectl apply -f /path/to/your/datadog-agent.yaml
   ```
 
+### Running Agents in a single container
+
+<div class="alert alert-warning">Available in Operator 1.4.0 or later</div>
+
+By default, the Datadog Operator creates an Agent Daemonset with pods running multiple Agent containers. Datadog Operator v1.4.0 introduces a configuration which allows users to run Agents in a single container. In order to avoid elevating privileges for all Agents in the single container, this feature is only applicable when `system-probe` or `security-agent` is not required. For more details check [privileges required by various Agents][7].
```

Comment:
```suggestion
By default, the Datadog Operator creates an Agent DaemonSet with pods running multiple Agent containers. Datadog Operator v1.4.0 introduces a configuration which allows users to run Agents in a single container. In order to avoid elevating privileges for all Agents in the single container, this feature is only applicable when `system-probe` or `security-agent` is not required. For more details, see [Running as an unprivileged user][7] on the Agent Data Security page.
```

Reply:
`DaemonSet` is what's used in the [Kubernetes docs](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) and most other pages in our own docs. 

Also updating the link text to be more descriptive per the [documentation style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#links).